_commands/basics/wc.md

```diff
@@ -2,23 +2,74 @@
 ---
 
 wc
--------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+--
+
+`wc` prints newline, word, and byte counts for each FILE, and a total if more than one FILE is specified. With no FILE, or when FILE is a dash ("-"), wc operates on standard input. (A word is a non-zero-length sequence of characters delimited by white space.)
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ wc [OPTION]... [FILE]...
+$ cd [OPTION]... --files0-from=F
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
 
-##### Break it down
+#### `wc`
+
+The `wc` command without passing any parameter will display a basic result of ”tecmint.txt file. The three numbers shown below are 12 (number of lines), 16 (number of words) and 112 (number of bytes) of the file.
```

Comment:
"tecmint.txt" can be surrounded by backticks. ex. `tecmint.txt`. See my previous comment about potentially moving this example to be the reference command example.